### ISSUE TITLE
No mention of localised ij-acute glyph

### ISSUE THREAD

**Issue by casperdewith** [MicrosoftDocs/typography-issues#996]
I browsed through [the chapter on OpenType font development](https://learn.microsoft.com/en-us/typography/opentype/). In it, some webpages of Polish and Icelandic people are cited, in which they suggest how to design certain letterforms. This shows that you care about proper localisation – fantastic! I would like to mention the language-specific rules of my language: Dutch.

Dutch needs this ij-acute ligature. Whenever `íj` (i-acute j) is typed in Dutch, acutes should appear on both the `i` and the `j`.
![ij-acute-piazzolla](https://user-images.githubusercontent.com/35010340/208203223-8cb0757f-8647-411c-976c-180b84076e69.png)

This is also true for the capital letters, where you can be even more creative. Something like this would be fantastic:
![image](https://user-images.githubusercontent.com/35010340/208204158-81df2408-18d1-46cc-8761-ab198d8f4a38.png)
This is the typeface [Livvic](https://github.com/Fonthausen/Livvic).

I think this rule deserves a place in your resource.

**Comment by moyogo** [comment]
> Whenever íj (i-acute j) is typed in Dutch, acutes should appear on both the i and the j.

This is not true. The j character should be combined with a U+0301 COMBINING ACUTE character.

The official Taalunie spelling rule, https://woordenlijst.org/leidraad/5/1, is clear (emphasis mine):
> Door technische beperkingen **vervalt meestal** het nadrukteken op de j van een lange ij. Bijvoorbeeld: blíjven kijken! 
>
> [= Because of technical limitations the stress mark is **usually omitted** from the j in the *lange ij*. For exampled: blíjven kijken!]

Before 1996, there was no standard rule on how many acutes digraphs composed of different letters should have. For example the majority of authors used *níet, móet, blíjf* with only the first letter of digraphs ie, oe, ij having an acute. See [Jan Renkema, *Schrijfwijzer*, 1987, p. 159](https://archive.org/details/schrijfwijzerhan0000renk/page/159/mode/1up) for example.

Text written with pre-1996 rules, text following the "usually omitted from the j" part of the official rule, or even mispelled texts should not have an acute added if it’s not there. For example the text of the spelling rule itself would be misrendered with a font that adds an acute to j after an í, or most texts written by users who learned to mark stress before 1996 would be misrendered.

**Comment by moyogo** [comment]
That said, a note about the Dutch ij and íj / íj́ (without and with COMBINING ACUTE) would be useful.
